spellbook/models/looksrare/ethereum/looksrare_ethereum_events.sql

```diff
@@ -154,4 +156,9 @@ LEFT JOIN  {{ ref('nft_ethereum_aggregators') }} agg ON agg.contract_address = t
 LEFT JOIN {{ source('prices', 'usd') }} p ON p.minute = date_trunc('minute', looks_rare.block_time)
     AND p.contract_address = currency_contract
     AND p.blockchain ='ethereum'
-LEFT JOIN {{ ref('tokens_ethereum_erc20') }} erc20 ON erc20.contract_address = currency_contract
\ No newline at end of file
+LEFT JOIN {{ ref('tokens_ethereum_erc20') }} erc20 ON erc20.contract_address = currency_contract
+WHERE tx.block_time > '2022-01-01'
```

Comment:
this may be more efficient as AND tx.block_time > '2022-01-01' under line 152 inner join constraints (but also same comment as above on date filter variable values vs. hardcoded)